.github/workflows/prs.yml

```diff
@@ -20,12 +17,12 @@ jobs:
         php-version: ${{ matrix.php }}
         extensions: pdo, sqlite, imagick
         coverage: none
-
     - name: Validate composer.json and composer.lock
       run: composer validate
-
     - name: Install dependencies
       run: composer install --prefer-dist --no-progress --no-interaction --no-suggest
+    - name: Run ECS
+      run: php vendor/bin/ecs
     - name: Run test suite
       run: |
         php vendor/bin/codecept build
```

Comment:
Is ECS > Codecept > PHPStan the order we really want for the execution of this job?

Reply:
IMO Yes; we need `codecept build` for phpstan to work on tests folder.
I considered splitting it but with the minimal size of this repo it's not worth it.